runtime/doc/ui.txt

```diff
@@ -800,6 +800,7 @@ must handle.
 		"rpc_error"	Error response from |rpcrequest()|
 		"return_prompt"	|press-enter| prompt after a multiple messages
 		"quickfix"	Quickfix navigation message
+		"search_echo"	Search echo message
```

Comment:
"search_cmd" seems more accurate, `SEARCH_ECHO` is just the name of the internal flag.
```suggestion
		"search_cmd"	Entered search command
```